app/Http/Controllers/BeatmapsetsController.php

```diff
@@ -283,6 +285,13 @@ public function update($id)
                         'new' => $beatmapset->nsfw,
                     ])->saveOrExplode();
                 }
+
+                if ($oldOffset !== $beatmapset->offset) {
```

Comment:
can use some tests (the test for others are missing as well but should at least add for this one)

Reply:
after thinking for a while I'm not sure what should i... test 🤔

Reply:
test if hitting the endpoint changes the value when it should (and doesn't when it shouldn't) and if it adds the event when it should (and doesn't when it shouldn't)